src/stories/Blocks/header/Header.tsx

```diff
@@ -109,21 +109,19 @@ export const Header = (props: HeaderProps) => {
                 ))}
               </ul>
             </div>
-            <div className="header__menu-profile header__button">
-              <a
-                href="/"
-                className="hide-linkstyle"
-                aria-label="user profile link"
-              >
-                {signedIn && haveNotification && (
-                  <div className="header__notification bg-signal-alert" />
-                )}
-                <ProfileIcon />
-                {signedIn && (
-                  <span className="text-small-caption">{username}</span>
-                )}
-              </a>
-            </div>
+            <button
```

Comment:
👏 Great that you're bringing the actual implementation back to the design system 👏